teaser/logic/buildingobjects/calculation/two_element.py

```diff
@@ -402,8 +402,9 @@ def calc_attributes(self):
                           ", please be careful with results. In addition " +
                           "this might lead to RunTimeErrors")
         self._sum_outer_wall_elements()
-        if len(self.thermal_zone.inner_walls) < 1:
-            warnings.warn('For thermal zone ' + self.thermal_zone.name,
+        if len(self.thermal_zone.inner_walls + self.thermal_zone.floors +
```

Comment:
This applies for all three changes:

I think your changes will work just fine. 

However if I have a look at the whole function it seems that `outer walls` and `windows` are treated different. In particular following code is missing for internal walls

```python
        for out_wall in outer_walls:
            out_wall.calc_equivalent_res()
            out_wall.calc_ua_value()
        for win in self.thermal_zone.windows:
            win.calc_equivalent_res()
            win.calc_ua_value()
``` 

In addition, the function `_sum_outer_wall_elements()` is called no matter if there are outer walls avaialbe or not. This will lead to errors if no outer wall is present (which shouldn't be the case any way, but still we should handle this, right?)

```python
        if len(outer_walls) < 1:
            warnings.warn("No walls are defined as outer walls for thermal " +
                          "zone " + self.thermal_zone.name + " in building " +
                          self.thermal_zone.parent.name +
                          ", please be careful with results. In addition " +
                          "this might lead to RunTimeErrors")
        self._sum_outer_wall_elements()
```

I propose to include these changes in this PR, as they are related